time.c

```diff
@@ -4371,16 +4377,19 @@ time_ceil(int argc, VALUE *argv, VALUE time)
 
 /*
  *  call-seq:
- *     time.sec -> integer
+ *    sec -> integer
+ *
+ *  Returns the integer second of the minute for +self+,
+ *  in range (0..60):
  *
- *  Returns the second of the minute (0..60) for _time_.
+ *    t = Time.new(2000, 1, 2, 3, 4, 5, 6)
+ *    # => 2000-01-02 03:04:05 +000006
+ *    t.sec # => 5
  *
- *  *Note:* Seconds range from zero to 60 to allow the system to inject
- *  leap seconds. See https://en.wikipedia.org/wiki/Leap_second for further
- *  details.
+ *  Note: the second value may be 60 when there is a
+ *  {leap second}[https://en.wikipedia.org/wiki/Leap_second].
  *
- *     t = Time.now   #=> 2007-11-19 08:25:02 -0600
- *     t.sec          #=> 2
+ *  Related: Time#year, Time#mon, etc.
```

Comment:
I think we can remove `etc.` from the list of related.  Ditto for other methods.

Reply:
I thought it useful to suggest that there's a complete family of such.

Reply:
Not my preference, but I don't object.